test/sql/types/list/list_null_members.test

```diff
@@ -30,55 +30,55 @@ SELECT lhs.i, rhs.i,
 FROM list_int lhs, list_int rhs
 ORDER BY 1, 2
 ----
-NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL	True	False
-NULL	[1]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-NULL	[1, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-NULL	[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-NULL	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-NULL	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-NULL	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1]	NULL	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1]	[1]	False	True	True	False	False	True	True	False
-[1]	[1, 2]	True	True	False	True	False	False	False	True
-[1]	[1, NULL]	True	True	False	True	False	False	False	True
-[1]	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1]	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1]	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, 2]	NULL	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, 2]	[1]	False	False	False	True	True	True	False	True
-[1, 2]	[1, 2]	False	True	True	False	False	True	True	False
-[1, 2]	[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, 2]	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, 2]	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, 2]	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, NULL]	[1]	False	False	False	True	True	True	False	True
-[1, NULL]	[1, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, NULL]	[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	True	False
-[1, NULL]	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, NULL]	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[1, NULL]	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL]	[1]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL]	[1, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL]	[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL]	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	True	False
-[NULL]	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL]	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, 2]	[1]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, 2]	[1, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, 2]	[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, 2]	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, 2]	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	True	False
-[NULL, 2]	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	[1]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	[1, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	[1, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	[NULL]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	[NULL, 2]	NULL	NULL	NULL	NULL	NULL	NULL	False	True
-[NULL, NULL]	[NULL, NULL]	NULL	NULL	NULL	NULL	NULL	NULL	True	False
+NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL	true	false
```

Comment:
Not a big thing, but can we restore the Title case for True/False to make the diff clearer?